app/src/main/java/org/mozilla/fenix/home/HomeMenu.kt

```diff
@@ -91,22 +92,15 @@ class HomeMenu(
     }
 
     private fun getSyncItemTitle(): String {
-        val authenticatedAccount = accountManager.authenticatedAccount
-        val email = accountManager.accountProfileEmail
-
-        return if (authenticatedAccount && !email.isNullOrEmpty()) {
-            email
-        } else {
-            context.getString(R.string.sync_menu_sign_in)
-        }
+        return accountManager.accountProfileEmail ?: context.getString(R.string.sync_menu_sign_in)
```

Comment:
Same as above, but I will also note the duplication of code between the 2. No idea, if we have a util class that happens to be used by both menus that we can throw this into, if not, don't worry about it for now.